### ISSUE TITLE
FH4

### ISSUE THREAD

**Issue by VoiceInMyHead** [cosmii02/RacingDSX#52]
FH4 is not in the list of applications in appcontainer. Why the mod doesn't work in the game. After launching the game with telemetry it did not appear in the list

**Comment by cosmii02** [comment]
Someone else had the issue as well and for the it went away by itself.. I haven't had the issue myself so I'm not sure what the solution is for now

**Comment by cosmii02** [comment]
@VoiceInMyHead Try the powershell command in the readme and let me know if that fixes it for you

**Comment by VoiceInMyHead** [comment]
Sorry, I don't understand what powershell command in the readme file is talking about.
While using this mod, I noticed that sometimes adaptive triggers work in FH4 without me using AppContainer. This happens by accident, they just turn on when I enter the game. I was unable to output the exact instructions for how to reproduce this.

**Comment by cosmii02** [comment]
within current testing fh4 works perfectly on 0.6.6 and the powershell command will be the way to do loopback going forward.

**Comment by tele-panda** [comment]
Same issue for me. I installed from Dodi repacks.
The event logs won't come on either the RacingDSX or the Live Incoming Data feed in DSX settings.
I set everything up correctly.

After installing it from steam(official version), it started working.
The problem might be that it could have been lacking updates or that cracked versions disable network input outputs usually.
This could probably be the issue with some people who come looking for a fix.

**Comment by cosmii02** [comment]
@tele-panda please try the powershell cmd listed in readme, that should work

**Comment by tele-panda** [comment]
> @tele-panda please try the powershell cmd listed in readme, that should work

I did try the powershell cmd as well, didn't work. Checked firewall too.

I don't care about the cracked version anymore.
They have online features disabled as usual. So I don't want to bother finding out how to undo the changes done by the people who cracked it and enable the network functionality. Or perhaps they changed the program identifier/name that your powershell cmd uses to enable network loopback. who knows.

Anyway, the official version from Steam is working👍as expected, I did use the powershell cmd for this and it worked.
I've given it a try past few days and it's incredibly good. It's very precise and enhances the understanding of how to use brake and throttle. ⭐
I've tried Adaptive trigger tools for other games like Assetto Corsa, and this is tool that gives the best experience of all.
